package.json

```diff
@@ -79,12 +80,12 @@
 		"@rollup/plugin-typescript": "^6.1.0",
 		"@sindresorhus/tsconfig": "^0.7.0",
```

Comment:
I reverted the update to this because it requires 109 changes in the code. It should be its own PR, maybe not one that just adds `!` after each